### PR TITLE
Add guidance for running lesson pilot workshops

### PR DESCRIPTION
**Pull request by tobyhodges**
This section provides guidance for community members who want to organise pilot workshops for a new lesson.

### CODE REVIEW

**Comment by maneesha** [comment]
@tobyhodges @ErinBecker  is this ready to merge?

File: topic_folders/lesson_development/lesson_pilots.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
+## Lesson Pilot Workshops
+
+### Purpose
+
+Teaching a lesson for the first time is very rewarding
+but the experience of the Instructors and Learners also
```
Comment (ErinBecker) [inline topic_folders/lesson_development/lesson_pilots.md:6]:
```suggestion
but the experience of the Instructors and learners also
```

File: topic_folders/lesson_development/lesson_pilots.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+and the requirements for piloting official Carpentries lessons,
+see [the Lesson Life Cycle chapter of The Carpentries Curriculum Development Handbook](https://cdh.carpentries.org/the-lesson-life-cycle.html).
+
+### Information for Lesson Developers
+
+#### Finding Hosts for Beta Pilots
+
+If you are developing a new official Carpentries lesson -
+a lesson developed based on prior agreement with The Carpentries,
+intended to become another lesson/curriculum offered
```
Comment (ErinBecker) [inline topic_folders/lesson_development/lesson_pilots.md:41]:
```suggestion
and which is intended to become another lesson/curriculum offered
```

File: topic_folders/lesson_development/lesson_pilots.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+in centrally-organised workshops -
+[the Curriculum Team](mailto:team@carpentries.org) will help you
+find hosts and Instructors for pilot workshops.
+
+If you are developing a lesson in The Carpentries Incubator,
+we encourage you to try to find hosts for pilot workshops yourself.
```
Comment (ErinBecker) [inline topic_folders/lesson_development/lesson_pilots.md:47]:
```suggestion
```

File: topic_folders/lesson_development/lesson_pilots.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+You can do this by putting out a call for hosts via
```
Comment (ErinBecker) [inline topic_folders/lesson_development/lesson_pilots.md:48]:
```suggestion
you can recruit pilot hosts by putting out a call via
```

File: topic_folders/lesson_development/lesson_pilots.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+helpful starting points.
+If after taking these steps,
+you find that you need help finding hosts to pilot your lesson,
+or if you have any questions about the lesson pilot process for
+lessons in The Carpentries Incubator,
+you can [contact incubator@carpentries.org](mailto:incubator@carpentries.org).
+
+#### Collecting Feedback on the Lesson
+
+Feedback from learners will be a valuable source of information
```
Comment (ErinBecker) [inline topic_folders/lesson_development/lesson_pilots.md:67]:
```suggestion
Feedback from learners will be a valuable source of information about
```

File: topic_folders/lesson_development/lesson_pilots.md [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+and suggestions about how your lesson could be further improved after the pilot.
```
Comment (ErinBecker) [inline topic_folders/lesson_development/lesson_pilots.md:68]:
```suggestion
and suggestions for how your lesson could be further improved after the pilot.
```

File: topic_folders/lesson_development/lesson_pilots.md [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+The standard Carpentries pre- and post-workshop surveys
+do not support lesson pilots
+so you will need to create your own surveys to send out before/after a pilot workshop.
+Although surveys for pilot workshops will frequently include questions
+that are specific to the particular lesson being trialled,
```
Comment (ErinBecker) [inline topic_folders/lesson_development/lesson_pilots.md:73]:
```suggestion
that are specific to the particular lesson being piloted,
```

File: topic_folders/lesson_development/lesson_pilots.md [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+This [template post-pilot workshop survey](https://docs.google.com/forms/d/1OGCQBotD2nOJkc7KpFZLhFfb3EBcxEDwHz_3p48qz3U/template/preview)
+can be copied and adapted to suit the needs of your lesson,
+and shared with learners in place of the standard post-workshop survey.
+
+It is also important to gather information about the lesson
+while it is being taught.
+Check
+[the Lesson Life Cycle chapter of The Carpentries Curriculum Development Handbook](https://cdh.carpentries.org/the-lesson-life-cycle.html#field-testing-alpha-stage)
+for a list of things to take note of during the pilot workshop.
+We recommend assigning a specific person or people with responsibility
```
Comment (ErinBecker) [inline topic_folders/lesson_development/lesson_pilots.md:85]:
```suggestion
We recommend assigning a specific person or people
```

File: topic_folders/lesson_development/lesson_pilots.md [hunk 9]
```diff
@@ -0,0 +1,125 @@
...
+to keep track of these points, e.g. an Instructor or Helper.
+
+### Information for Hosts
+
+#### Recruiting Instructors for Beta Pilots
+
+If you are hosting a pilot of a new official Carpentries lesson -
+a lesson developed based on prior agreement with The Carpentries,
+intended to become another lesson/curriculum offered
```
Comment (ErinBecker) [inline topic_folders/lesson_development/lesson_pilots.md:94]:
```suggestion
and which is intended to become another lesson/curriculum offered
```

File: topic_folders/lesson_development/lesson_pilots.md [hunk 10]
```diff
@@ -0,0 +1,125 @@
...
+[the Curriculum Team](mailto:team@carpentries.org) will help you
+find Instructors for pilot workshops.
+
+The Carpentries is also keen to support the development and piloting
+of lessons in The Carpentries Incubator.
+If you are hosting a pilot of a lesson in the Incubator,
+we ask that you **first** try to find Instructors for pilot workshops yourself.
+Often, hosts are able to recruit certified Instructors from their local
+community
+with relevant knowledge of the lesson topic
```
Comment (ErinBecker) [inline topic_folders/lesson_development/lesson_pilots.md:105]:
```suggestion
with relevant knowledge of the lesson topic,
```

File: topic_folders/lesson_development/lesson_pilots.md [hunk 11]
```diff
@@ -0,0 +1,125 @@
...
+but in some cases this will not be possible.
+If you wish to recruit Instructors for a pilot workshop,
+try putting a call out on
+[local/regional community mailing lists](https://carpentries.topicbox.com/groups),
+any relevant channels on [The Carpentries Slack workspace](https://carpentries.org/connect/) (the lesson authors may be able to direct you to these),
+and/or [by publishing a post on our blog](https://docs.carpentries.org/topic_folders/communications/guides/submit_blog_post.html).
+Please do not post calls for Instructors to the general channel on Slack, or the discuss and instructors lists on TopicBox: any messages to recruit Instructors will be removed from those channels. If after taking these steps, you find that you need help finding Instructors for your lesson pilot, you can [contact incubator@carpentries.org](mailto:incubator@carpentries.org)
```
Comment (ErinBecker) [inline topic_folders/lesson_development/lesson_pilots.md:112]:
```suggestion
Please do not post calls for Instructors to the general or instructors channel on Slack, or the discuss and instructors lists on TopicBox: any messages to recruit Instructors will be removed from those channels. If after taking these steps, you find that you need help finding Instructors for your lesson pilot, you can [contact incubator@carpentries.org](mailto:incubator@carpentries.org)
```

File: topic_folders/lesson_development/lesson_pilots.md [hunk 12]
```diff
@@ -0,0 +1,125 @@
...
+for assistance.
+
+#### Creating a Pilot Workshop Webpage
+
+[The Carpentries workshop webpage template](https://github.com/carpentries/workshop-template)
+supports the creation of webpages for pilot workshops.
+See [the Customisation page of the template documentation](https://carpentries.github.io/workshop-template/customization/#configuration-file-_configyml)
+for details of how to configure the webpage for a pilot workshop.
```
Comment (ErinBecker) [inline topic_folders/lesson_development/lesson_pilots.md:120]:
```suggestion
for instructions on how to configure the webpage for a pilot workshop.
```

File: topic_folders/lesson_development/lesson_pilots.md [hunk 13]
```diff
@@ -0,0 +1,125 @@
...
+
+The Carpentries does not currently track pilot workshops in the same
```
Comment (ErinBecker) [inline topic_folders/lesson_development/lesson_pilots.md:122]: We do track pilots for official-track lessons. They should be reported as self-organized workshops using the standard form. (Sher and I will work together to make sure the curriculum that is being piloted gets added to the form before pilots start.)

**Review by ErinBecker — CHANGES_REQUESTED** [review]
The only required change is my last comment about tracking official-track pilots. Everything else is minor language suggestions. 😄

File: topic_folders/lesson_development/lesson_pilots.md (hunk 13 above)
Comment (tobyhodges) [reply topic_folders/lesson_development/lesson_pilots.md]: Thanks for catching that inaccuracy, @ErinBecker. I have corrected in a0f2a56

**Comment by tobyhodges** [comment]
All of @ErinBecker's comments and suggestions have been addressed now and this can be merged @maneesha

**Pull request merged**